src/components/layout.tsx

```diff
@@ -5,9 +5,7 @@ import { Navbar } from "./navbar/navbar";
 const Layout: React.FC = ({ children }) => {
   return (
     <div className="w-screen min-h-screen pt-20">
-      <h1 className="text-center tracking-widest text-4xl md:text-8xl">
-        SpaceX Space Center
-      </h1>
+      <h1 className="text-center tracking-widest">SpaceX Space Center</h1>
```

Comment:
Now that I think about it, I should also retrieve `text-center tracking-widest` back to our global style. What do you think?

Reply:
Yep, we could do that indeed.